package/MDAnalysis/lib/_cutil.pyx

```diff
@@ -65,6 +78,187 @@ def unique_int_1d(np.ndarray[np.int64_t, ndim=1] values):
             is_monotonic = False
     result = result[:j + 1]
     if not is_monotonic:
-        result.sort()
-        result = unique_int_1d(result)
-    return result
+        result = unique_int_1d(np.sort(result))
+
+    return np.array(result)
+
+
+cdef intset difference(intset a, intset b):
```

Comment:
I think @kain88-de tried to use this but it's not available on osx or something

Reply:
We can rewrite the algorithm ourselves. They assume that the containers are sorted (which they are for c++11 and later).  Using the std algorithm is difficult on linux. I did not get the compiler to resolve the template. This could get better when we update to use the new conda compilers.